spatialmath/geometry.go

```diff
@@ -15,6 +15,8 @@ type Geometry interface {
 	Transform(Pose) Geometry
 	ToProtobuf() *commonpb.Geometry
 	CollidesWith(Geometry) (bool, error)
+	// If DistanceFrom is negative, it represents the penetration depth of the two geometries, which are in collision.
+	// Penetration depth magnitude is defined as the minimum translation which would result in the geometries not colliding.
```

Comment:
Could we add the disclaimer on this function that for certain entity pairs we do a conservative estimate?  I feel like we should be advertising that this is something of an experimental feature at the moment

Reply:
Yes, will do